xml/System.Net.WebSockets/ValueWebSocketReceiveResult.xml

```diff
@@ -102,8 +102,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets whether the message has been received completely.</summary>
+        <value><see langword="true"/> if the message was received completely; otherwise, <see langword="false"/>.</value>
```

Comment:
minor nit to avoid having the CI touching this later.

```suggestion
        <value>
          <see langword="true"/> if the message was received completely; otherwise, <see langword="false"/>.</value>
```

Reply:
@mairaw Why does this avoid triggering CI?  It seems counterintuitive that an opening tag should be on a separate line and closing tag on the same line; I would expect either all on one line or both opening and closing tags on separate lines.